dojo.py

```diff
@@ -84,11 +131,6 @@ def do_add_person(self, args):
         name = str(fname)+" "+str(lname)
```

Comment:
You don't have to convert to string. Every user entry will always be a string which you can then cast to another type.